requirements.txt

```diff
@@ -5,5 +5,8 @@ isort
 flake8
 tox
 python-openstackclient
+pyOpenSSL
```

Comment:
I believe this is already in `setup.py`. I think the time has come to write a `test-requirements.txt` which captures the requirements for gating purpose. We can pick this in another PR though. Just a thought.

Reply:
I will list down these in seperate test-requirements.txt in a seperate PR.